ls1intum/Artemis
# `General`: Add documentation for team exercises

### Checklist
#### General

### Motivation and Context

Add page about `Team exercises` to documentation.

### Description

Added a new page for `Team exercises`, and added links to it in the Readme and in the About us page.

Reviews would be appreciated on, particularly on if I should go more in depth in some aspects, if I missed anything important, and where the page should go (I wasn't too sure where to put it but decided to put it alongside the differente exercise types).

### Steps for Testing

* Ensure that the link to the documentation page works correctly on the About Us page and in the Readme
* Checkout the new documentation page (link above)

### Review Progress


#### Code Review
#### Manual Tests
